patroni/tags.py

```diff
@@ -89,3 +89,8 @@ def nosync(self) -> bool:
     def replicatefrom(self) -> Optional[str]:
         """Value of ``replicatefrom`` tag, if any."""
         return self.tags.get('replicatefrom')
+
+    @property
+    def nostream(self) -> bool:
+        """``True`` if ``nostream`` is ``True``, else ``False``."""
+        return bool(self.tags.get('nostream', False))
```

Comment:
I guess it was copied from `nosync` or similar, but behavior isn't really nice, because it treats any non-empty strings like `True`.
We probably better to use something like:
```suggestion
        return parse_bool(self.tags.get('nostream')) or False
```

The reason we didn't change other methods - just to be backward compatible in case if someone has configuration file in a weird format. But for the new methods we can handle it nicely.

P.S. validator.py is OK, it could be stricter.

Reply:
> The reason we didn't change other methods - just to be backward compatible in case if someone has configuration file in a weird format. But for the new methods we can handle it nicely.

oh, thats explains it

Reply:
done